.github/workflows/tests.yml

```diff
@@ -129,7 +129,17 @@ jobs:
         run: mage build
 
       - name: Install tarantool
-        run: brew install tarantool
+        run: |
```

Comment:
Please set up Python at L120 and remove the line with Python installation.

Reply:
changed